### PR TITLE
Updating changelog.md to the our published release notes for 0.8.2

### PR DESCRIPTION
**Pull request by kolloch**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -37,22 +37,44 @@ Health checks now work with HTTPS.
 
 #### Faster (configurable) task distribution
 
-In order to speed up task launching and use the resource offers Marathon receives from Mesos more efficiently,
-we added a new offer matching algorithm, that tries to start as many tasks as possible per task offer cycle.
-The maximum numbers of tasks to start are configurable with the follwing startup parameters:
+Mesos frequently sends resource offers to Marathon (and all other frameworks). Each offer will represent the available resources of a single node in the cluster. Until now, Marathon would only start a single task per resource offer, which led to slow task launching in smaller clusters. In order to speed up task launching and use the resource offers Marathon receives from Mesos more efficiently, we added a new offer matching algorithm, that tries to start as many tasks as possible per task offer cycle. The maximum numbers of tasks to start are configurable with the following startup parameters:
```
Comment (philipnrmn) [inline changelog.md]: Mesos frequently sends resource offers to Marathon (and all other frameworks). Each offer will represent the available resources of a single node in the cluster. Before this change, Marathon would only start a single task per resource offer, which led to slow task launching in smaller clusters. In order to speed up task launching and use the resource offers Marathon receives from Mesos more efficiently, we added a new offer matching algorithm which tries to start as many tasks as possible per task offer cycle. The maximum number of tasks to start is configurable with the following startup parameters:

File: changelog.md [hunk 2]
```diff
@@ -37,22 +37,44 @@ Health checks now work with HTTPS.
...
 
 `--max_tasks_per_offer_cycle` (default 1000): The maximum number of tasks to start in total per cycle
 
+**Example**
+
+Given a cluster with 200 nodes and the default settings for task launching. If we want to start 2000 tasks, it would take at least 10 cycles, because we are only starting 1 task per offer, leading to a total maximum of 200. If we change the `max_tasks_per_offer` setting to 10, we could start 1000 tasks per offer (the default setting for `max_tasks_per_offer_cycle`), reducing the necessary cycles to 2. If we also adjust the `max_tasks_per_offer_cycle ` to 2000, we could start all tasks in a single cycle (given we receive offers for all nodes).
+
+**Important**
+
+Starting too many tasks at once can lead to a higher number of status updates being sent to Marathon than it can currently handle. We will improve the number of events Marathon can handle in a future version. A maximum of 1000 tasks has proven to be a good default for now. `max_tasks_per_offer ` should be adjusted so that `NUM_MESOS_SLAVES * max_tasks_per_offer == max_tasks_per_offer_cycle `. E.g. in a cluster of 200 nodes it should be set to 5.
```
Comment (philipnrmn) [inline changelog.md]: `max_tasks_per_offer ` contains a hanging space

**Comment by philipnrmn** [comment]
A few small suggestions in that first change, otherwise :shipit:

**Comment by kolloch** [comment]
Cool, thanks.

**Pull request merged**